lib/CloudinaryPlugin.js

```diff
@@ -239,27 +212,20 @@ class CloudinaryPlugin {
    * 
    * If one of both is missing, a bad request error is thrown.
    * 
-   * Several public_ids can be given but it must be send in a comma-separated string format
+   * Several public_ids can be given in the public_id arg.  
    * Be careful of white space as Cloudinary allows space in the public_ids
    * 
    * @param {Request} request 
    * 
    * @see https://cloudinary.com/documentation/image_upload_api_reference#tags_method
    */
   async add_tag(request) {
-    const tag = this._getArg(request, 'tag'), to = this._getArg(request, 'public_ids').split(",");
-    return await Cloudinary.uploader.add_tag(tag, to)
+    const tag = this._getArg(request, 'tag'), to = this._getArrayArg(request, 'public_id');
+    const res = await Cloudinary.uploader.add_tag(tag, to);
```

Comment:
If two variable are already defined with a unique `const` keyword, why do you define another variable with a new `const`?